README.md

```diff
@@ -18,55 +18,31 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
+- run the command `git clone https://github.com/Elisha-Tetteyfio/To-do-list.git` in your command line 
+- switch to the project directory
+- Enjoy ;)
```

Comment:
- [ ] After cloning the repo the users need to set up the app as well by installing packages and running the server. Kindly include step-by-step instructions on how to run the project.